### PR TITLE
Document the mail.templates_basepath setting.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

File: production.ini [hunk 1]
```diff
@@ -126,6 +126,11 @@ use = egg:bodhi-server
 # domain for generated message IDs
 # message_id_email_domain = admin.fedoraproject.org
 
+# Where to search for e-mail templates. Eash release has an attribute that defines the name of a
```
Comment (sebwoj) [inline production.ini]: Just small typo: Eash

**Review by sebwoj — CHANGES_REQUESTED** [review]

File: production.ini (hunk 1 above)
Comment (bowlofeggs) [reply production.ini]: Good catch, fixed!

File: production.ini [hunk 2]
```diff
@@ -126,6 +126,11 @@ use = egg:bodhi-server
 # domain for generated message IDs
 # message_id_email_domain = admin.fedoraproject.org
 
+# Where to search for e-mail templates. Each release has an attribute that defines the name of a
```
Comment (pypingou) [inline production.ini]: Would it be worth documenting which attribute that is?

**Review by pypingou — COMMENTED** [review]
One question, otherwise looks fine :)

File: production.ini (hunk 2 above)
Comment (bowlofeggs) [reply production.ini]: @pypingou I'm applying this patch to document that attribute:

```
diff --git a/production.ini b/production.ini
index b2e57d9e..891bd1f8 100644
--- a/production.ini
+++ b/production.ini
@@ -126,8 +126,9 @@ use = egg:bodhi-server
 # domain for generated message IDs
 # message_id_email_domain = admin.fedoraproject.org

-# Where to search for e-mail templates. Each release has an attribute that defines the name of a
-# mail template, and Bodhi will look in the path defined by this setting to find the named template.
+# Where to search for e-mail templates. Each release has a mail_template attribute that defines the
+# name of a mail template, and Bodhi will look in the path defined by this setting to find the named
+# template.
 # mail.templates_basepath = bodhi:server/email/templates/
```

Comment (pypingou) [reply production.ini]: Looks good to me, thanks :)

**Comment by bowlofeggs** [comment]
This patch is planned for inclusion in the upcoming 4.0.0 release: #3221

**Comment by bowlofeggs** [comment]
Bodhi 4.0.0 beta is built and deployed to staging:

https://copr.fedorainfracloud.org/coprs/bowlofeggs/bodhi-pre-release
https://bodhi.stg.fedoraproject.org/

**Pull request merged**